sycl/include/sycl/info/aspects.def

```diff
@@ -52,3 +52,5 @@ __SYCL_ASPECT(ext_oneapi_ballot_group, 54)
 __SYCL_ASPECT(ext_oneapi_fixed_size_group, 55)
 __SYCL_ASPECT(ext_oneapi_opportunistic_group, 56)
 __SYCL_ASPECT(ext_oneapi_tangle_group, 57)
+__SYCL_ASPECT(ext_oneapi_is_composite, 58)
+__SYCL_ASPECT(ext_oneapi_is_component, 59)
```

Comment:
Just to quickly double check... I'm surprised that these new aspects aren't also marked "experimental" in some way, but I can see that there are already other experimental aspects in this list.  @steffenlarsen, is this what you'd expect?

Reply:
I believe this is the norm. In general I don't believe we mark the interfaces themselves as experimental, only the namespace. Another case would be when an experimental extension adds a new member to an existing SYCL class, we don't mark those as experimental either. I suppose there are pros and cons to that approach, but I think the biggest pro is that promoting them means only changes to the namespace, which should hopefully make transitioning over to the promoted version smoother.

Reply:
Ok, that makes sense.  And we can't change the namespace here because `sycl::aspect` is an existing `enum`.